requirements/base.txt

```diff
@@ -1,17 +1,17 @@
 chevron~=0.12
-click~=7.0
-Flask~=1.0.2
+click~=7.1
```

Comment:
Python will install 7.1 automatically. Do we need a bump here?

Reply:
We don't need to, but keeping our requirement up to date looks better.
I put all these no-effect upgrade under this commit:
https://github.com/aws/aws-sam-cli/pull/2305/commits/db9f0221e3ff7db0fbe7e9de9d6504299dfe4212
chore: Update versions in in base.txt (no effective difference)

Reply:
The point in defining it this way to so that when pip installs happen, patch versions get installed.

Reply:
`~=7.0` means the `7.x.y` can be installed, what the difference when I changed it to `~=7.1`?